Update release guidance (#7084)
alphagov/govuk-frontend: docs/releasing/after-publishing-a-release.md

- any guidance updates are published and show as expected
- any updates coming from GOV.UK Frontend macro options show as expected

## Send an email to mailing list subscribers

We usually send out an email with the release comms, except for releases with minor fixes.

This email should:

- give a summary of the update
- explain why teams should update
- say which components or styles are affected (if applicable)
- thank any major contributors using their GitHub usernames

A release email will typically end with a link to the version's [release notes](https://github.com/alphagov/govuk-frontend/releases), and a call-to-action button to the [update npm how-to page](https://frontend.design-system.service.gov.uk/updating-with-npm/#update-using-node-js-package-manager-npm).

For example, see the [release email for GOV.UK Frontend 5.11.0](https://mailchi.mp/c877961d1db5/feature-release-govuk-frontend-v590-improved-file-upload-component-10977163?e=[UNIQID]).

## Send Slack messages to relevant channels

> We should update both internal and cross-government Slack channels:
> 
> - [GDS: govuk-design-system](https://gds.slack.com/archives/CAF8JA25U)
> - [GDS: digital-service-platforms](https://gds.slack.com/archives/C01E20X06JK)
> - [x-gov: govuk-design-system](https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6)

Reviewer: **suggestion** Just noticed that this list is no longer up to date. @calvin-lau-sig7 the list is usually in the comms draft, would it be OK to say something like:

```suggestion
We should update both internal and cross-government Slack channels listed with the Slack message draft.
```

That'd be easier to maintain than having the list both here and in the comms drafts, possibly drifting appart with time.

We can fix that in a subsequent PR, though 😊

Author: Lets do it as a follow up just so we can get the main improvements in and build from there?
